src/background.js

```diff
@@ -21,15 +21,26 @@ function init() {
   browser.alarms.onAlarm.addListener(handleWake);
   browser.notifications.onClicked.addListener(handleNotificationClick);
   browser.runtime.onMessage.addListener(handleMessage);
-  handleWake();
+  browser.storage.local.get().then(items => {
```

Comment:
This feels like a big enough change already.  How about we do that in a followup pr?  :)